ElementX/Sources/Services/Timeline/TimelineItems/Items/Messages/TextRoomTimelineItemContent.swift

```diff
@@ -19,4 +19,6 @@ import UIKit
 struct TextRoomTimelineItemContent: Hashable {
     let body: String
     var formattedBody: AttributedString?
+    // The textual representation of the formatted body (usually HTML code)
```

Comment:
Not suggesting to word it exactly like this, but I find this helpful clarify the meaning for me:
```suggestion
    // The original textual representation of the formatted body directly from the event (usually HTML code)
```